packages/integration-tests/history/test/index.js

```diff
@@ -38,18 +38,7 @@ const t = {
     return Object.assign({}, objectConfiguration.example)
   },
   renewSequelize: function() {
-    this.sequelize = new Sequelize('testdb', null, null, {
-      dialect: "sqlite",
-      storage: sqlitePath,
-      define: {
-        charset: 'utf8',
-        dialectOptions: {
-          collate: 'utf8_general_ci'
-        }
-      },
-      sync: { force: true },
-      logging: null // comment this line to show Sequezile logs in console
-    })
+    this.sequelize = new Sequelize('sqlite::memory:')
```

Comment:
Charset options are needed because by default it's different and tests should test setup close to production. It's enough to set `storage: ":memory:"` according to https://sequelize.org/master/manual/dialect-specific-things.html

And leave all other options intact.